src/__tests__/utils/repository-key-consistency.test.ts

```
@@ -0,0 +1,155 @@
+      const key = createRepositoryBranchKey('my-repo', 'main');
+      expect(key).toBe('my-repo:main');
+    });
+
+    it('should handle develop branch', () => {
+      const key = createRepositoryBranchKey('my-repo', 'develop');
+      expect(key).toBe('my-repo:develop');
+    });
+
+    it('should throw error when repository contains colon', () => {
+      expect(() => createRepositoryBranchKey('my:repo', 'main')).toThrow(
+        'Repository name cannot contain colons: "my:repo"'
```

ESLint

[error] Insert `,`

(prettier/prettier)